CONTRIBUTING.md

```diff
@@ -188,6 +188,9 @@ When editing documentation, use the following standards to demonstrate the examp
    Example: `**+l**\ *label*` results in **+l***label*.
 3. Optional arguments are placed wrapped with [ ] (square brackets).
 4. Arguments that are mutually exclusive are separated with a | (bar) to denote "or".
+5. Double-quotes are used to indicate a string value. Example: "05m". An argument
```

Comment:
As mentioned in https://github.com/GenericMappingTools/pygmt/issues/983#issuecomment-807224037:

> Another alternative way is using bold text, for example, **05m**, as bold text means literal arguments in both GMT and PyGMT documentation.

This standard recommend double quotes for strings ("05m"), but some GMT arguments use bold text. For example, the `verbose` parameter can accept strings like "w", "d" and others, and we use bold strings following the GMT convention (see https://www.pygmt.org/dev/api/generated/pygmt.Figure.basemap.html#pygmt.Figure.basemap). 

Will it cause more confusion?

Reply:
GMT and PyGMT both use bold for literal arguments in the html documentation, but GMT uses different formatting for the command line documentation. I think this is the main downside of using the formatting standards from GMT's RST documentation in the python docstrings and where using bold inside strings could get confusing. For example, here is the python console help for the `verbose` parameter in Figure.basemap()
<img width="572" alt="image" src="https://user-images.githubusercontent.com/14077947/113008351-f4745380-9144-11eb-821e-1fdc8dc8b8d6.png">
relative to the command line documentation for the **-V** option in GMT's basemap module:
<img width="558" alt="image" src="https://user-images.githubusercontent.com/14077947/113008503-166dd600-9145-11eb-8fbf-d01857b6f16d.png">

Reply:
Actually, I am fine with bold strings. I think Python users know whether it is a string and how to indicate a string value.

Reply:
I'm still hesitate to adopt either style: **bold** or "double-quotes".

Reply:
> I'm still hesitant to adopt either style: **bold** or "double-quotes".

Why? Or what do you prefer?

Reply:
We have to make a choice. Let's adopt **bold** texts.

Reply:
I understand using bold text for stand-along strings (for example, the list of verbose options that you originally shared), but when it is within a code block I think it needs to be double quotes. For example, do a search for single quotes in this latest commit https://github.com/GenericMappingTools/pygmt/commit/e131d26f9666bb28aa11d938f232f30036791976 and you will see that most are part of a code block inside docstrings.

Reply:
> when it is within a code block I think it needs to be double quotes.

Yes, I agree.

Reply:
> 2. Literal GMT arguments should be **bold** by wrapping the arguments with \*\*
   (two asterisks) on both sides. The argument description should be in *italicized*
   with \* (single asterisk) on both sides.

If I understand it correctly, we all agree that this rule also applies to string-type arguments like **05m**, right? If yes, we probably don't need to add a new rule, but extend this rule by adding one more example.

Reply:
Add an example in 6af2deb.